openlibrary/coverstore/warc.py

```diff
@@ -284,10 +164,10 @@ def write(self, record):
         """Writes a record into the WARC file.
         Assumes that data_length and other attributes are correctly set in record.header.
         """
-        self.file.write(str(record.get_header()).encode('utf-8'))
+        self.file.write(str(record.get_header()))
```

Comment:
This comment may also still apply
https://github.com/internetarchive/openlibrary/pull/3389/files#r418155774